src/context.rs

```diff
@@ -21,20 +21,17 @@ use crate::timer;
 /// game's window.
 ///
 /// Most functions that interact with the hardware, for instance
-/// drawing things, playing sounds, or loading resources (which then
-/// need to be transformed into a format the hardware likes) will need
-/// to access the `Context`.  It is an error to create some type that
+/// drawing things, playing sounds, or loading resources will need
+/// to access one, or rarely even two, of its sub-contexts.
+/// It is an error to create some type that
 /// relies upon a `Context`, such as `Image`, and then drop the `Context`
-/// and try to draw the old `Image` with the new `Context`.  Most types
-/// include checks to make this panic in debug mode, but it's not perfect.
```

Comment:
I removed this as it's no longer true in this branch. Do we want to re-add this functionality?

Reply:
Is it even possible to drop the context from within the event loop?

Reply:
I don't think so, but technically you could create, for example, an `Image` before starting the loop, run the loop, terminate it and then create another context and put that `Image` into a new related `EventHandler`, allowing use of it in the new context, right?

But that sounds like an error that is very obvious and therefore doesn't need such safe-guards to prevent it to me.